build_optee_libraries.sh

```diff
@@ -41,7 +41,7 @@ export CROSS_COMPILE64="aarch64-linux-gnu-"
 
 # build optee_os and optee_client for qemu_v8
 git clone https://github.com/OP-TEE/optee_os.git -b $OPTEE_VERSION $OPTEE_DIR/optee_os
-(cd $OPTEE_DIR/optee_os && make PLATFORM=vexpress-qemu_armv8a)
+(cd $OPTEE_DIR/optee_os && make PLATFORM=vexpress-qemu_armv8a CFG_TA_FLOAT_SUPPORT=n)
```

Comment:
I am not sure the reason for this change. 
But we have another `build_optee_libraries.sh` in `scripts/setup` for update, if this change is necessary.

Reply:
Please see the commit message:
```
When building tls_client and tls_server 32bit TAs, this error occurred:
  multiple definition of `__aeabi_fcmple' (`__aeabi_fcmpeq' and others)
This means the __aeabi functions are defined both in Rustc
compiler_builtins and optee libutils.

Disable the CFG_TA_FLOAT_SUPPORT when building libutils is a easy
approach without porting the compiler_builtins.
```

Since disable the float support in compiler_builtins requires manual porting, I think it's the easiest way to fix the 32bit TA building error. Seems our current TAs don't need the float support, it works for now. If there're new TAs need the float support, we can handle it then.

If it's acceptable I will update the one in `scripts/setup`.

Reply:
Thanks for outlining the underlying issue, and I agree with the proposed actions. 
I suggest inlining the reason for this change directly as a comment in the script for future maintenance, rather than keeping it only in the commit message. This will make it clearer for anyone reading the build scripts later.